src/pytest_elasticsearch/factories.py

```diff
@@ -26,11 +26,20 @@
 from pytest_elasticsearch.port import get_port
 
 
+def return_config(request):
+    """Return a dictionary with config options."""
+    config = {}
+    config['logsdir_option'] = request.config.getoption('logsdir')
```

Comment:
let's have this function already solve the order in which we read and override the previous. Ie here we should only leave one key in the dictionary: `logsdir`